lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +20,19 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(text, str):
+        return None
+
+    text = text.lower()
+
+    letters_list = []
+
+    for i in text:
+        if i.isalpha():
```

Comment:
use continue operator if element of text is not alpha

Reply:
fixed